api/early-access_API-Filtering.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Advanced API Filtering (Preview)
+sidebarTitle: Advanced API Filtering
+description: Learn how to use Advanced API to make operational tooling faster and more precise.
+tag: Preview
+---
+
+<Info>
+This feature is under active development. Behavior, supported fields, and limits may change before General Availability (GA). This guide is provided as forward-looking context for evaluation and feedback.
+</Info>
+
+
+Advanced API filtering helps you quickly surface the resources that matter, using a server-side subset of the Common Expression Language (CEL) to evaluate expressions against each resource instance before results are returned. Using Advanced API Filtering helps avoid downloading large collections and filtering client-side, and helps make operational tooling faster and more precise.
+
+**Request shape**
+
+```http
+GET /{resource}?filter='{CEL_EXPRESSION}'
+```
+
+Example:
+
+```http
+GET /endpoints?filter='obj.type == "cloud" || obj.type == "agent"'
+```
+
+---
+
+## Supported CEL (subset)
+
+The following core operators and helpers are supported for the initial release:
+- Logical operators: `!`, `&&`, `||`
+- Comparative operators: `<`, `<=`, `==`, `!=`, `>=`, `>`
+- Parentheses for grouping
+- List membership using the `in` keyword
+- String substring checks: `startsWith()`, `contains()`, `endsWith()`
+- Length / emptiness checks: `size()`, `== ""`, `== null`
+- Date parsing: `datetime.parse()`
+
+### Instance Inspection (vs. List Comprehension)
+
+Expressions are evaluated against a single **resource instance** exposed as `obj`. Compare fields **on the instance** rather than attempting list-wise checks on fields.
+
+✅ Valid
+```http
+GET /endpoints?filter='obj.type == "cloud" || obj.type == "agent"'
+GET /endpoints?filter='obj.type in ["agent", "cloud"]'
+```
+
+❌ Not valid
+```http
+GET /endpoints?filter='["agent","cloud"] in obj.types'
+```
+
+---
+
+## Dates and time helpers
+
+- **Treat timestamps as numerics** by using `<`, `<=`, `==`, `>=`, `>` directly on timestamp fields.  
+- **Parsing:** `datetime.parse("<ISO-8601>")` to compare against string literals.  
+- **Convenience helpers:** `now()` and `daysAgo(n)` support concise relative filters, e.g.:
+  ```http
+  GET /endpoints?filter='obj.created_at >= daysAgo(7)'
+  ```
+
+---
+
+## Query restrictions and limitations
+
+### Unsupported CEL features (initial release)
+
```

Comment:
```suggestion

To keep filter evaluation small and predictable, the following CEL features will be excluded in the initial release of Advanced API Filtering.

```

I think it's worth moving this explanation above the list. I also reworded it to mention that it's an initial release thing, and put the reasoning first